### ISSUE TITLE
stray smart quote in Makefile

### ISSUE THREAD

**Issue by brion** [jscodec/jsvpx#5]
```
$ make
rm -rf build 
mkdir build  
VERSION=`node -pe "require('./package.json').version"` && \
	browserify src/main.js -t [ envify purge --MODE global ] | \
	tee  build/flare-vpx-“$VERSION".js 
/bin/sh: -c: line 0: unexpected EOF while looking for matching `"'
/bin/sh: -c: line 1: syntax error: unexpected end of file
make: *** [build-release] Error 2
```

**Comment by brion** [comment]
It occurs to me the Makefile looks like it's been replaced by the webpack build; should it be removed?

**Comment by brianxautumn** [comment]
Yes, originally started with a makefile, but then started using npm scripts cause easier. But now if the scripts start getting longer, it might make more sense to just use a separate make or build script again. Maybe remove for now?

**Comment by brianxautumn** [comment]
Actually it does make more sense to use makefile. Going move the build scripts to there. Just added a new one to build the ogv example

**Comment by brion** [comment]
My one warning on using a Makefile is that it's harder to make a local build that works on both Unix and Windows.

For ogv.js I'm planning to split out the emscripten-built modules, which require a Unix build infrastructure for all the autoconf libraries anyway, and avoid Makefiles on the front-end build in favor of Grunt.

**Comment by brianxautumn** [comment]
That is a very good point. Npm for now it is!
